app/models/dialog_field_tag_control.rb

```diff
@@ -40,10 +40,6 @@ def self.allowed_tag_categories
     categories.sort_by { |tag| tag[:description] }
   end
 
-  def self.category_tags(category_id)
```

Comment:
With the UI side PR, yes. I found this method was used on the UI side, but it's kind of useless because it was instantiating a new `DialogFieldTagControl` with the given category id and then calling values, but the values method needs to take into consideration things like sort order, required, and data_type. I've replaced all of the explicit calls of `DialogFieldTagControl.category_tags` with either the [new helper function that does something similar](https://github.com/ManageIQ/manageiq-ui-classic/pull/694/files#diff-89b5547fb78cd2fc8378ed0769a9603fR12) or a [helper that modifies `field.values`](https://github.com/ManageIQ/manageiq-ui-classic/pull/694/files#diff-89b5547fb78cd2fc8378ed0769a9603fR7)